content/operate/kubernetes/deployment/helm.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: Install Redis Enterprise Helm chart
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: Install the Redis Enterprise for Kubernetes version 7.8.Wisconsin using helm charts.
+linkTitle: Helm
+weight: 11
+---
+
+Helm charts provide a simple way to install the Redis Enterprise for Kubernetes operator in just a few steps. For more information about Helm, go to [https://helm.sh/docs/](https://helm.sh/docs/).
+
+{{<note>}} This feature is currently in public preview and is not supported on production workloads. Only new installations of the Redis operator are supported at this time. The steps for [creating the RedisEnterpriseCluster (REC)]({{<relref "operate/kubernetes/deployment/quick-start/#create-a-redis-enterprise-cluster-rec">}}) and other custom resources remain the same.{{</note>}}
+
+## Prerequisites
+
+- A [supported distribution]({{< relref "/operate/kubernetes/reference/supported_k8s_distributions.md" >}}) of Kubernetes.
+- At least three worker nodes.
+- [Kubernetes client (kubectl)](https://kubernetes.io/docs/tasks/tools/).
+- [Helm 3.10 or later](https://helm.sh/docs/intro/install/).
+
+## Install
+
+1. Add the `redis` repository.
+
+    ```sh
+    helm repo add redis-enterprise https://helm.redis.io/
+    ```
+
+1. Install the Helm chart into a new namespace.
+
+    ```sh
+    helm install <operator-name> redis/redis-enterprise-operator \
+        -- version <release-name> \
+        -- namespace <namespace-name> \
+        -- create-namespace
+    ```
+
+To install with Openshift, add `--set openshift.mode=true`.
```

Comment:
I'm definitely not a language authority, but is it "install **with** OpenShift" or "install **to** OpenShift" (or "install to an OpenShift cluster")?
Also - I believe it's Open**S**hift and not Open**s**hift.